src/components/tooltip/Tooltip.tsx

```diff
@@ -73,27 +73,27 @@ export interface TooltipProps {
     /** A CSS class for the tooltip */
     className?: string;
     /** Whether to constrain the tooltip to the element's scroll parent. Defaults to `false` */
-    constrainToScrollParent: boolean;
+    constrainToScrollParent?: boolean;
```

Comment:
These props are all required, but have default values set. Thinking about making them optional (due to the defaults).